sandbox/vm/BUILD

```diff
@@ -19,32 +19,61 @@ ts_library(
     ],
 )
 
-ts_library(
-    name = "compile_loader",
-    srcs = [],
-    data = [
-        ":compile_loader.js",
-    ],
+load("//packages:index.bzl", "pkg_bundle", "pkg_npm_tar")
+
+externals = [
+    "vm2",
+    "glob",
+    "protobufjs",
+    "protobufjs/minimal",
+]
+
+genrule(
+    name = "copy_vm2",
+    srcs = ["//:node_modules/vm2"],
+    outs = ["node_modules/vm2"],
+    cmd = "cp -rp $(SRCS) $(OUTS)",
+    output_to_bindir = 1,
+)
+
+genrule(
+    name = "copy_glob",
+    srcs = ["//:node_modules/glob"],
+    outs = ["node_modules/glob"],
+    cmd = "cp -rp $(SRCS) $(OUTS)",
+    output_to_bindir = 1,
+)
+
+genrule(
+    name = "copy_protobufjs",
+    srcs = ["//:node_modules/protobufjs"],
+    outs = ["node_modules/protobufjs"],
+    cmd = "cp -rp $(SRCS) $(OUTS)",
+    output_to_bindir = 1,
+)
```

Comment:
Interested in a tidier way of doing this; needs to be done in external bazel repos too